javascript/javascript2/week2/doublingOfNumbers.js

```diff
@@ -0,0 +1,11 @@
+
+function getDoubleOddNumber () {
+  let result = doubledOddNumber;
+  return result
+}
+
+const numbers = [1, 2, 3, 4];
+const oddNumbers = numbers.filter (number => number % 2);
+let doubledOddNumber = oddNumbers.map(x => x*2)
+
+console.log("The doubled numbers are", getDoubleOddNumber());
```

Comment:
This is fine, but since `getDoubleOddNumber()` only returns `doubledOddNumber`, this line can be written as 
    console.log("The doubled numbers are", doubledOddNumber);